static-assets/components/cstudio-common/resources/es/base.js

```diff
@@ -687,7 +687,8 @@ CStudioAuthoring.Messages.registerBundle("contentTypes", "es", {
     publishStarted:"Se ha iniciado el proceso de publicación.",
 
     /*Datasources */
-    childContent: "Contenido Hijos",
+    childContent: "Contenido Hijo",
+    embeddedContent: "Embedded Content",
```

Comment:
Not translated. Change to "Contenido Empotrado"

Reply:
Doesn't seem like contentType was added to this translations file. Also, still missing the other languages.

Reply:
2: It is already there on the translations.

Reply:
updated.

Reply:
Added Korean and German translations.